admin/class-admin-init.php

```diff
@@ -59,7 +59,10 @@ public function enqueue_dismissible() {
 	 * Redirect first time or just upgraded users to the about screen.
 	 */
 	public function after_update_notice() {
-		if ( current_user_can( 'manage_options' ) && ! $this->seen_about() ) {
+
+		$can_access = is_multisite() ? WPSEO_Utils::grant_access() : current_user_can( 'manage_options' );
```

Comment:
I think it would make sense to rewrite `grant_acess()` implementation more clearly, will try to.